pkg/sql/sqlbase/structured.proto

```diff
@@ -434,6 +434,9 @@ message DescriptorMutation {
   optional uint32 mutation_id = 5 [(gogoproto.nullable) = false,
       (gogoproto.customname) = "MutationID", (gogoproto.casttype) = "MutationID"];
   reserved 6;
+
+  // Indicates that this mutation has been rolled back.
+  optional bool rollback = 7 [(gogoproto.nullable) = false];
```

Comment:
bikeshed nit: does it indicate that it is a rollback or the it has been *rolled* back?

Reply:
it is a rollback